packages/frontend/src/components/Explorer/CustomMetricWriteBackModal/index.tsx

```diff
@@ -21,6 +21,60 @@ import MantineIcon from '../../common/MantineIcon';
 import { useWriteBackCustomMetrics } from './hooks/useCustomMetricWriteBack';
 import { usePreviewWriteBackCustomMetrics } from './hooks/usePreviewCustomMetricWriteBack';
 
+const CreatedPullRequestModalContent = ({
```

Comment:
extracted all the modal content from CreatedPullRequestModalContent into a new component, to use it in both single and multiple